src/Components/TodoList.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface TodoListPropsType {
+  todosToShow: Todo[],
+}
+
+export const TodoList: React.FC<TodoListPropsType> = ({ todosToShow }) => {
+  return (
+    <section className="todoapp__main">
+      {todosToShow.map(todo => (
+        <div
+          key={todo.id}
+          className={classNames(
+            'todo',
+            { completed: todo.completed },
+          )}
+        >
+          <label className="todo__status-label">
+            <input
+              type="checkbox"
+              className="todo__status"
+              checked
```

Comment:
The `checked` attribute value for the input checkbox should be assigned to `todo.completed` instead of being set as a static value. Change it to `checked={todo.completed}`.